text/conversations/x_labyrinth/x6081_cnpcfathom_oasis_callistege.stringtable

```diff
@@ -1105,7 +1105,7 @@ Plötzlich und gleichzeitig erlöschen ihre Echos, was sie allein mitten in eine
       <ID>197</ID>
       <DefaultText>Callisteges Hand erscheint auf dem Glas und wird dann schnell weggeschnappt. Ein knirschender, metallischer Missklang steigt zu einem Crescendo, was die Wände des Labyrinths und die Glastür der Höhle schüttelt, bis er schließlich abklingt.
 
-Die Glastür schiebt sich offen und einer tiefschwarze Staub geißt wie Wein heraus.
+Die Glastür schiebt sich offen und tiefschwarzer Staub gießt wie Wein heraus.
```

Comment:
man könnte auch sagen: schiebt sich auf

Reply:
da hast du recht - hatte den ersten Satzteil wohl nicht bemerkt. soll ich ein neues commit schicken?

Reply:
Ja, bitte.

Reply:
Anpassung erfolgt in commit:
https://github.com/AurelioSilver/tidesofnumenera-german-patch/pull/10/commits/a52322bca3e5e8f8bf0c2c305418622400b7f102

Passt so?

Reply:
Ja, danke.